config.toml

```diff
@@ -20,10 +20,12 @@ otherConfig = 'Other install media'
 centOS = 'CentOS 7'
 
 [remotes]
-defaultRemote = '8565fc53-ea32-40ea-b380-dd1b8f3b5d44'
+defaultRemote1 = '8565fc53-ea32-40ea-b380-dd1b8f3b5d44'
```

Comment:
It should be specific to the scenario. If we want to change it for a scenario it should not impact other tests.

Reply:
You can ignore the above commentary let's do simple. Let's provide `defaultRemote`, `NFS remote`,  `SMB remote`.

Reply:
Let's replace ids by the remote entries.